compiler/damlc/daml-opts/daml-opts/DA/Daml/Options.hs

```diff
@@ -68,6 +71,21 @@ toCompileOpts options@Options{..} reportProgress =
                 then Just path
                 else Nothing
       | otherwise = pure Nothing
+    -- NOTE(MH): The type class contexts used with generic templates sometimes
+    -- trigger the `Opt_WarnSimplifiableClassConstraints` warning, which
+    -- suggests to use the `MonoLocalBinds` language extension. However, this
+    -- extensions does no play well with our record-dot-syntax. Thus, we
+    -- suppress these warnings.
+    isGenTemplMonoLocal msg
+        | Just (_, clasz:rest) <- stripInfix ["The", "constraint"] (T.words msg)
```

Comment:
What does the `z` in `clasz` stand for? Surely the name is not z-encoded? :slightly_smiling_face: